config/secrets.yml.template

```diff
@@ -1,18 +1,20 @@
 # Sample secrets.yml file.
-# Top-level key is secret name, second-level key is environment name.
-# (`base` is an alias for staging, test, levelbuilder, and production.)
-# The value is the secret string.
-# (Complex objects will be converted to JSON.)
 ---
-foo:
-  base: bar # staging, test, levelbuilder and production
-  production: baz
-qux:
-  development:
-    qux: quux
-  staging:
-    qux: quuz
-  test:
-    qux: quuz
-  production:
-    qux: quuz
+# Publish AWS Secrets Manager Secrets named `[environment type]/cdo/acme_api_key` that can be used to populate `CDO.acme_api_key`
+acme_api_key:
+  development: 123ABC      # Development API key
+  adhoc: 123ABC            # Development API key
+  staging: 123ABC          # Development API key
+  test: ZYX987             # Test API key
+  levelbuilder: ZYX987     # Test API key
+  production: a1b2c3d4e5f6 # Production API key
+
+# Publish AWS Secrets Manager Secrets named `[environment type]/cdo/wile_e_coyote_credentials` that can be used to
+# populate `CDO.wile_e_coyote_credentials` with a value whose data type is complex.
```

Comment:
I know this is already merged, but in case there is a chance for a followup: The issue we ran into with Statsig seemed to be that we weren't sure how to verify the AWS secrets looked 'right'. Could that be in a comment somewhere here? i.e. something like 'Once these secrets appear on AWS secret manager, you can verify the formatting of the secret 'value' is the plaintext string, not a nested object'

Reply:
I think that belongs in secrets.md, but perhaps something like this?

```
Cdo::Secrets.new.get!('development/cdo/your_new_secret')
```